.gitignore

```diff
@@ -7,6 +7,8 @@ __pycache__
 deploy.cfg
 .coverage
 test/test_temp/
+src/data_lakehouse_ingest/minio_utils.py
+src/data_lakehouse_ingest/spark_session.py
```

Comment:
Would you be able to add these to the repo so that everything that people need is in one place, and people could set this up locally (e.g. with docker containers) or on another Spark provider (e.g. NERSC) and run the code?